sdk/core/Azure.Core/src/DynamicData/DynamicData.cs

```diff
@@ -46,7 +46,8 @@ internal static JsonSerializerOptions GetSerializerOptions(DynamicDataOptions op
             {
                 Converters =
                 {
-                    new DefaultTimeSpanConverter()
+                    new DefaultTimeSpanConverter(),
+                    new BinaryDataConverter()
```

Comment:
Tip: always use a trailing comma whenever you can and this diff would've been just a single line. Can also help avoid some merge conflicts.
```suggestion
                    new BinaryDataConverter(),
```